update-ubuntu-or-debian-server/README.md

```
@@ -1,4 +1,570 @@
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](../LICENSE)
+[![Last Commit](https://img.shields.io/github/last-commit/bigbeartechworld/big-bear-scripts)](https://github.com/bigbeartechworld/big-bear-scripts/commits/master)
+
+# 🚀 BigBear Ubuntu/Debian Server Update Script v2.0.0
+
+## 📑 Table of Contents
+
+- [Features](#-features)
+  - [Core Functionality](#-core-functionality)
+  - [Safety & Security](#️-safety--security)
+  - [Monitoring & Reporting](#-monitoring--reporting)
+  - [Configuration Management](#-configuration-management)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)